x-pack/test_serverless/shared/services/svl_user_manager.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SAMLSessionManager } from '@kbn/test';
+import { FtrProviderContext } from '../../functional/ftr_provider_context';
+
+export function SvlUserManagerProvider({ getService }: FtrProviderContext) {
+  const config = getService('config');
+  const log = getService('log');
+  const isCloud = !!process.env.TEST_CLOUD;
+  // Sharing the instance within FTR config run means cookies are persistent for each role between tests.
+  const sessionManager = new SAMLSessionManager(config, log, isCloud);
```

Comment:
I'm not sure if Cypress test runner has access to `config` instance, so I image a possible alternative like:
```ts
new SAMLSessionManager(kbnHost, auth, log, isCloud)
```

where `kbnHost` and `auth` FTR and Cypress runner should provide on its own:
```ts
const kbnHost = {
    protocol: config.get('servers.kibana.protocol'),
    hostname: config.get('servers.kibana.hostname'),
    port: isCloud ? undefined : config.get('servers.kibana.port'),
};
const auth = { username: config.get('servers.kibana.username')}, password: config.get('servers.kibana.password') }
```

@MadameSheema @pheyos wdyt?

Reply:
I believe the alternative is the best one for Cypress usage. Thanks :)

Reply:
Changed in 82f750b027c14b67999105c0a8c44c5722660fae